Gemfile.lock

```diff
@@ -96,6 +96,8 @@ GEM
       nanoc (~> 4.9)
     nenv (0.3.0)
     netrc (0.11.0)
+    nokogiri (1.13.4-arm64-darwin)
```

Comment:
It appears changes to the Gemfile shouldn't be committed

Reply:
Given someone decided in the past to commit Gemfile.lock rather than adding it to .gitignore, I think we can assume that if it changes, we should commit the change.

Reply:
I'm not going to create an issue to add `Gemfile.lock` to `.gitignore` because it's used as a snapshot of all the required gems and their dependencies. By committing `Gemfile.lock` the bundler doesn't have to recalculate all the gem dependencies each time the project is run. This file would have updated itself when I ran this project locally.